src/styles/utils/variables.scss

```diff
@@ -0,0 +1,17 @@
+// stopwatch========================================================================================================================================================
```

Comment:
These comments are redundant. It should be clear from variable's name what it is